e2e/nx-run/src/run.test.ts

```diff
@@ -582,7 +582,7 @@ describe('Nx Running Tests', () => {
         const output = runCommand('npm run build', {
           cwd: pkgRoot,
         });
-        expect(output).toContain('Nx read the output from the cache');
+        expect(output).toContain('All -1 commands built from cache!');
```

Comment:
The new output here doesn't make sense to me... All negative 1 tasks?

Reply:
Ah, yes, sorry if this was unclear - I'd meant to use a draft to indicate it's not yet ready for review.